### PR TITLE
[MIG][15.0] website_sale_stock: migration script

### PR DESCRIPTION
**Pull request by sang250399**
This PR
======
Migration done

### CODE REVIEW

**Comment by daonamutc1** [comment]
Check everything is done, waiting for depend.

File: openupgrade_scripts/scripts/website_sale_stock/15.0.1.0/post-migration.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+            END
+        """,
+    )
+    openupgrade.logged_query(
+        env.cr,
+        """
+            UPDATE product_template
+            SET show_availability = CASE
+            WHEN inventory_availability = 'threshold' THEN true ELSE false
+            END
```
Comment (CarlosRoca13) [inline openupgrade_scripts/scripts/website_sale_stock/15.0.1.0/post-migration.py:21]:
```suggestion
            UPDATE product_template
        SET allow_out_of_stock_order = CASE
            WHEN inventory_availability IN ('never', 'custom') THEN true ELSE false END,
        show_availability = CASE
            WHEN inventory_availability IN ('always', 'threshold') THEN true ELSE false END,
        available_threshold = CASE WHEN inventory_availability = 'always' THEN 999999999 END
```

You can do it in one query and coverage all options from older versions

**Review by CarlosRoca13 — CHANGES_REQUESTED** [review]

**Comment by MiquelRForgeFlow** [comment]
Superseded by https://github.com/OCA/OpenUpgrade/pull/3725.

**Pull request closed without merging**